File: php/newton-raphson-method.php
<?php 
        // Formula: Xn+1 = Xn + C/Xn

        $X0 = 2;
        $C = 5;

        echo "Formula: Xn+1 = Xn + C/Xn\n";
        echo "X0: $X0\n";

        for ($n = 1; $n < 11; $n++) {
            $Xn = ($X0 + ($C / $X0)) / 2;
            if ($Xn == $X0) {
                break;
            } else {
                echo "X$n: $Xn\n";
                $X0 = $Xn;
            }
        }
?>

File: c++/newton-raphson-method.cpp
#include <iostream>
using namespace std;

int main() {
    // Formula: Xn+1 = Xn + C/Xn
    float X0 = 2;
    float C = 5;
    cout << "Formula: Xn+1 = Xn + C/Xn\n";
    cout << "X0: " << X0;

    for (int n = 1; n < 12; n++) {
        float Xn = (X0 + (C / X0)) / 2;
        if (Xn == X0) {
            break;
        } else {
            cout << "\nX" << n << ": " << Xn;
            X0 = Xn;
        }

    }
    cout << "\n";
    return 0;
}